src/types/presentation_time.zig

```diff
@@ -0,0 +1,66 @@
+const wlr = @import("wlroots.zig");
+
+const wayland = @import("wayland");
+const wl = wayland.server.wl;
+
+const Presentation = extern struct {
+    pub const Event = extern struct {
```

Comment:
I think we should move this out of the namespace of `Presentation` and make it just `wlr.PresentationEvent`